react/features/speaker-stats/functions.ts

```diff
@@ -163,14 +168,14 @@ function getEnhancedStatsForOrdering(state: IReduxState, stats: any, orderConfig
 /**
  * Filter stats by search criteria.
  *
- * @param {IReduxState} state - The redux state.
- * @param {Object | undefined} stats - The unfiltered stats.
+ * @param {IState} state - The redux state.
+ * @param {ISpeakerStats | undefined} stats - The unfiltered stats.
```

Comment:
is this really returning as well `undefined`? eg: when no criteria and stats, it's returning default value from `getSpeakerStats` which is empty object